src/cryptonote_core/service_node_list.cpp

```diff
@@ -994,6 +994,12 @@ namespace service_nodes
       return false;
     }
 
+    if (hf_version >= cryptonote::network_version_19 && stake.transferred < get_min_node_operator_contribution(info.staking_requirement))
+    {
+      LOG_PRINT_L1("Register TX rejected: TX does not have sufficient operator stake");
+      return false;
+    }
+
```

Comment:
This check is in the wrong place: it should be done as part of service_node_rules.cpp's `check_service_node_portions(...)` where we check all the other portions.  For hf >= 19 it can special-case the `i == 0` case to use `get_min_node_operator_contribution` rather than `get_min_node_contribution` and check all contribution requirements in one go/one place.  (And then once that check is there, this one is unnecessary because we'll have already ensured that `reserved >= required`, and the previous `if` statement assures `transfered >= reserved`.